src/Bicep.LangServer/Snippets/Templates/res-automation-module.bicep

```diff
@@ -0,0 +1,13 @@
+﻿// Automation Module
+resource automationAccount_automationVariable 'Microsoft.Automation/automationAccounts/modules@2015-10-31' = {

```

Comment:
TODO: I will update it to use parent-child syntax:
https://github.com/Azure/bicep/blob/main/docs/spec/resource-scopes.md#parent-child-syntax

Reply:
Updated here:
https://github.com/Azure/bicep/commit/ac634afda59d439b448f45ffdc33c5d25f592fad